src/api/itemPublish.ts

```diff
@@ -61,7 +63,14 @@ export const getItemPublishedInformation = async (
     .get<ItemPublished>(
       `${API_HOST}/${buildGetItemPublishedInformationRoute(id)}`,
     )
-    .then(({ data }) => data);
+    .then(({ data }) => data)
+    .catch((error) => {
+      if (error.status === StatusCodes.NOT_FOUND) {
+        // this means that the collection is not published, we return undefined in this case
+        return undefined;
+      }
+      throw error;
```

Comment:
I had the same pb with ItemGeolocation. 
I preferred changing the backend to return null with 200 because NOT_FOUND could be trigger because the item is not found as well.
What do you think?

Reply:
Ah yes I see, in this case we can change the backend if it is a better solution.